crates/database/src/migrations/sql/V8__add_files.sql

```diff
@@ -0,0 +1,18 @@
+-- Files table to track OpenAI files per user
+CREATE TABLE files (
+    id VARCHAR(255) PRIMARY KEY, -- OpenAI file ID
+    user_id UUID NOT NULL REFERENCES users(id) ON DELETE CASCADE,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
```

Comment:
Should we track file names as well so it's easier to list and browse all the files owned by the user?